docs/administering/ssl.md

```diff
@@ -1,29 +1,38 @@
 ## Sandstorm and HTTPS
 
-If you are using a hostname like `example.sandcats.io`, then you likely already have working HTTPS
-(SSL) for your hostname. This page provides details on a variety of options for setting up HTTPS,
-including the `sandcats.io` free certificates.
+Sandstorm can now terminate TLS connections for most conventional configurations you may wish to employ.
```

Comment:
I think we should leave out the word "now" here; the fact that this didn't used to be the case isn't super relevant to folks reading these docs (as opposed to release notes or such).

Reply:
Agreed